extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/enhancer/HibernateEntityEnhancerTest.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.hibernate.orm.enhancer;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Can you avoid the star imports? We are trying to avoid them.

Other than that, it looks good, nice catch.

Reply:
Good catch. Done